db/seeds.rb

```diff
@@ -1,9 +1,21 @@
-# db/seeds.rb
-
 require "faker"
 
+# Don't send emails during seed process.
+ActionMailer::Base.perform_deliveries = false
+
+# Clear existing data
+LogbookEntry.destroy_all
+Aircraft.destroy_all
+User.destroy_all
+Notification.destroy_all
+
+USERS_COUNT = 10
+AIRCRAFT_NOTIFICATIONS_COUNT = 1000
+LOGBOOK_ENTRY_NOTIFICATIONS = 1000
+USER_NOTIFICATIONS = 100
+
 # Create Users
-10.times do
+users = USERS_COUNT.times do
```

Comment:
```suggestion
users = USERS_COUNT.times.map do
```

need to initialize `users` as an `Array` so that we can call `.sample` on it